cwms-data-api/src/main/java/cwms/cda/data/dto/timeseriesprofile/ParameterInfo.java

```diff
@@ -36,14 +36,26 @@ public String getUnit() {
         return unit;
     }
 
-    public int getIndex() {
+    public Integer getIndex() {
         return index;
     }
 
-
+    public Integer getStartColumn(){
+        return startColumn;
+    }
+    public Integer getEndColumn(){
+        return endColumn;
+    }
     @Override
     public void validate() throws FieldException {
-
+        if(index==null && !(startColumn!=null && endColumn!=null))
```

Comment:
I may need to pull down my logic design book but I'm pretty sure the second check is a perfect application for the "exclusive or"

Reply:
But let someone else besides me decide if the logic makes sense. Doing too much boolean algebra homework makes you overthink these things.

Reply:
my intention was to mimic in the expression that if index is null, then start and end must not be null,

Reply:
Oh yeah, the intention is at least clear, just very... nested.
Though I just roughed out a truth table and a clearer way didn't immediately present itself. I think xor would work in aggregate but you wouldn't be able to have as clear of an error message as you currently do. So just ignore me on this one.